specification/compute/resource-manager/Microsoft.Compute/GalleryRP/stable/2025-03-03/gallery.json

```diff
@@ -2966,6 +3607,248 @@
       ],
       "description": "The definition of a parameter that can be passed to a custom action of a Gallery Application Version."
     },
+    "GalleryScript": {
+      "type": "object",
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/GalleryScriptProperties"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../common-types/v1/common.json#/definitions/Resource"
+        }
+      ],
+      "description": "Specifies information about the gallery Script Definition that you want to create or update."
+    },
+    "GalleryScriptUpdate": {
+      "type": "object",
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/GalleryScriptProperties"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/UpdateResourceDefinition"
+        }
+      ],
+      "description": "Specifies information about the gallery Script Definition that you want to update."
+    },
+    "GalleryScriptProperties": {
+      "type": "object",
+      "properties": {
```

Comment:
>eula [](http://example.com/codeflow?start=9&length=4)

Do you expect the whole agreement to go here ?

Reply:
yes. It's been used by other major Compute Gallery resources since many years
https://msazure.visualstudio.com/One/_git/Compute-CPlat-PIR?path=/src/CAPS/Service/API/Model/CapsApiModelCommon.cs&version=GBdevelop&line=56&lineEnd=57&lineStartColumn=1&lineEndColumn=1&lineStyle=plain&_a=contents